docs/manuals/run-deploy/deploy/azure.md

```diff
@@ -1,4 +1,83 @@
 # Deploy on Azure
 
-TODO: [taipy-doc#397](https://github.com/Avaiga/taipy-doc/issues/397)
+[Azure] (https://azure.microsoft.com/en-us), one of the world's leading providers of cloud services, is the perfect platform for deploying Taipy applications. It allows developers to stay focused on their application bringing effortless scalability, monitoring and security natively. The following documentation shows how to deploy on Azure App Service and get an application into the hands of the end-user.
 
+## Prerequisites
+
+- Minimal knowledge of Azure.
+- Azure CLI should be installed. Check [the official documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for azure CLI installation.
+- [:material-arrow-right: Running a Taipy application](../run/index.md)
+
+
+## Azure App Service
+
+[Azure App Service](https://learn.microsoft.com/en-us/azure/app-service/) is a managed platform that simplifies web application deployment, scaling, and management. Azure Web App, a service within App Service, specializes in hosting and managing web applications. As a Python developer, you can leverage Azure Web App to quickly deploy and run your Python-based web applications in a scalable, hassle-free environment.
+
+!!! Note
+
+    You can create Azure App Service and Azure Web App using Azure portal, VS Code, Azure Tools extension pack, or Azure CLI. Here we will focus on Azure CLI.
+
+## Prepare your application
+
+Open a terminal and go in the application folder. Ensure you have a `requirements.txt` file listing the required dependencies and your entry point Python file named `app.py` or `application.py`. These files are crucial for Azure App Service to deploy your application correctly.
+
+Because Azure Web Application runs applications with [Gunicorn](https://gunicorn.org/), your application must be adapted to be startable by Gunicorn. To do so, you must expose an `app` object of type `Flask` as shown in the following example.
+
+Here is the folder structure and content:
+```shell
+taipy@taipy:~$ ls
+application.py  requirements.txt
+taipy@taipy:~$ cat requirements.txt
+taipy
+taipy@taipy:~$ cat application.py
+import taipy as tp
+
+tp_app = tp.Gui(page="# Getting started with *Taipy*")
+
+if __name__ == "__main__":
+   # Development mode, Flask runs the application for debugging.
+   tp_app.run()
+else:
+   # Production mode, Azure Web Application runs the application with Gunicorn.
+   app = tp_app.run(run_server=False)
+```
+
+## Create a Web App in Azure
+
+In your terminal, log in to Azure using the command:
+```shell
+taipy@taipy:~$ az login
+```
+
+Still in your terminal, you can create the web app and then deploy your code using the following command:
```

Comment:
"Still in your terminal, you can create the web app"
->
"You can now create the web application"